package.json

```diff
@@ -2,18 +2,136 @@
   "name": "wp-dev-lib",
   "version": "1.0.0",
   "description": "Common code used during development of WordPress plugins and themes",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
   "repository": "xwp/wp-dev-lib",
+  "engines": {
+    "node": ">=6.11.x"
+  },
   "keywords": [
     "wordpress",
     "plugins",
     "themes",
     "development",
     "tools"
   ],
-  "author": "Weston Ruter <weston@xwp.co> (https://xwp.co)",
+  "author": {
+    "name": "Weston Ruter",
+    "email": "weston.ruter@xwp.co",
+    "url": "https://www.xwp.co"
+  },
+  "contributors": [
+    {
+      "name": "Piotr Delawski",
+      "email": "piotr.delawski@xwp.co"
+    },
+    {
+      "name": "Mike Crantea",
+      "email": "mike.crantea@xwp.co"
+    },
+    {
+      "name": "Justin Kopepasah",
+      "email": "justin.kopepasah@xwp.co",
+      "url": "https://kopepasah.com"
+    }
+  ],
   "license": "MIT",
-  "homepage": "https://github.com/xwp/wp-dev-lib#readme"
+  "homepage": "https://github.com/xwp/wp-dev-lib#readme",
+  "scripts": {
+    "build:self": "./build-workflows.sh",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "browserslist": [
+    "last 2 Chrome versions",
+    "last 2 Firefox versions",
+    "last 2 Safari versions",
+    "last 2 Edge versions",
+    "last 2 Opera versions",
+    "last 2 iOS versions",
+    "last 1 Android version",
+    "last 1 ChromeAndroid version",
+    "ie 11",
+    "> 1%"
+  ],
+  "babel": {
+    "presets": [
+      [
+        "env"
+      ]
+    ]
+  },
+  "stylelint": {
+    "extends": "stylelint-config-wordpress/scss",
+    "plugins": [
+      "stylelint-no-unsupported-browser-features"
+    ],
+    "rules": {
+      "string-quotes": "single",
+      "color-hex-length": null,
+      "function-parentheses-space-inside": null,
+      "function-url-quotes": "always",
+      "at-rule-empty-line-before": [
+        "always",
+        {
+          "except": [
+            "first-nested",
+            "blockless-after-same-name-blockless"
+          ],
+          "ignore": [
+            "after-comment"
+          ],
+          "ignoreAtRules": [
+            "else",
+            "elseif"
+          ]
+        }
+      ],
+      "plugin/no-unsupported-browser-features": [
+        true,
+        {
+          "severity": "warning"
+        }
+      ],
+      "selector-pseudo-element-colon-notation": "double",
+      "value-keyword-case": null
```

Comment:
@ntwb Thank you for your comment.

First of all, please note that this ruleset is still a work in progress and may change.

To answer your question - we decided to relax or override some of the rules defined in the `stylelint-config-wordpress/scss` based on the way we usually format our SCSS code.

In some points our best practices align quite well with the CSS rules set by Automattic in the [Calypso project](https://github.com/Automattic/wp-calypso/blob/master/.stylelintrc):

* We prefer using single quotes (`string-quotes`). It reflects the way strings in PHP and JS are treated in WordPress.
* We always use quotes, in `url` functions too (`function-url-quotes`).
* We allow whitespace inside parentheses (`function-parentheses-space-inside`). It, again, better reflects the way we format PHP and JS code.
* We are not requiring the hex color values to be short whenever possible (`color-hex-length`), since they will get optimized/shortened in the SCSS compilation process.

There are also grey areas, not defined in the WordPress CSS Coding Standards, e.g.:
* Since the standards describe CSS and not SCSS code, we decided to set the `at-rule-empty-line-before` rule in this particular way. We feel that the code is easier to read with the such setup.
* The standards don't mention the `selector-pseudo-element-colon-notation`. Our guess is that the requirement for a single colon is included in the `stylelint-config-wordpress` to maintain the Internet Explorer 8 support. Since IE 8 is no longer supported by WordPress as of version 4.8, we feel our code should use the valid CSS notation, i.e. double colons.

When it comes to the `value-keyword-case` my opinion is that is should not be overridden here. We should always use lowercase for CSS values. @mehigh and @kopepasah what do you think about it?